### DISCUSSION TITLE
How do I set a custom color?

### DISCUSSION THREAD

**Discussion by TheCheddarCheese** [fastfetch-cli/fastfetch#1159] [Q&A]
Putting a hex code in the color field doesn't seem to work. What do I do?

**Comment by CarterLi** [comment]
`38;2;r;g;b`

**Reply by JannisHeydemann** [reply]
what does the 38 and the 2 stand for?

**Reply by CarterLi** [reply]
https://en.m.wikipedia.org/wiki/ANSI_escape_code#24-bit
